assets/js/blocks/price-filter/block.js

```diff
@@ -27,22 +54,31 @@ import './style.scss';
  * @param {boolean} props.isEditor Whether in editor context or not.
  */
 const PriceFilterBlock = ( { attributes, isEditor = false } ) => {
+	const filteringForPhpTemplate = getSetting(
+		'is_rendering_php_template',
+		''
+	);
+
 	const [ minPriceQuery, setMinPriceQuery ] = useQueryStateByKey(
 		'min_price',
-		null
+		findGetParameter( 'min_price' ) * 100 || null
 	);
 	const [ maxPriceQuery, setMaxPriceQuery ] = useQueryStateByKey(
 		'max_price',
-		null
+		findGetParameter( 'max_price' ) * 100 || null
 	);
 	const [ queryState ] = useQueryStateByContext();
 	const { results, isLoading } = useCollectionData( {
 		queryPrices: true,
 		queryState,
 	} );
 
-	const [ minPrice, setMinPrice ] = useState();
-	const [ maxPrice, setMaxPrice ] = useState();
+	const [ minPrice, setMinPrice ] = useState(
+		findGetParameter( 'min_price' ) * 100 || null
+	);
+	const [ maxPrice, setMaxPrice ] = useState(
+		findGetParameter( 'max_price' ) * 100 || null
```

Comment:
Since these statements are used twice, what do you think of refactoring them above? Something like:

```js
const startingMinPrice = findGetParameter( 'min_price' ) * 100 || null;
// ...
```